### DISCUSSION TITLE
[Keyball44]左Ctrl、e、d、cキーが押しっぱなしになっています

### DISCUSSION THREAD

**Discussion by don-wang** [Yowkees/keyball#461] [Q&A] [answered]
あけましておめでとう。
Keyball44を組み立てて、動作確認したところ、左側のCtrl、e、d、cキーが押しっぱなしになっています。
![PXL_20231231_151138579](https://github.com/Yowkees/keyball/assets/1377797/7b2cab0a-d467-45c3-92cc-62bb6e9a4a5b)

USBを繋ぐと、この状態になって、他の操作ができません。ただし、OLED上、それ以外のキーは反応することを確認しました。

スイッチとソケットを外しても変わらないです。今基盤の様子をご確認ください。
![PXL_20231231_150706261](https://github.com/Yowkees/keyball/assets/1377797/7204e6d5-9107-4b93-981f-2480bc462647)

確認すべきダイオードなど、原因特定の方法を教えていただきたいです。

よろしくお願いします。

**Comment by Yowkees** [comment, marked as answer]
ご連絡ありがとうございます。

E含む縦一列のキーに不具合があるということで、これらに共通する信号線はCOL3になります。

お写真を見ますと、下記画像の緑丸部がショートしているようですので、これを除去してみてください。
ちょうどこの端子がCOL3ですので、これでおそらく直ると思います。

以上、お手数ですがご対応お願いいたします。
![KB44_col3fix2](https://github.com/Yowkees/keyball/assets/84209772/032953af-5048-4e4d-9828-ea3fe504471d)

**Reply by don-wang** [reply]
@Yowkees
ご返答ありがとうございます。
こちらがCOL3か、基盤上の表記のままと勘違いました。
ご指摘した通り、半田し直したら解決できいました。
これからKeyball44楽しみします。
